modules/oe_content_publication/config/post_updates/00003_create_fields/field.storage.node.oe_publication_contact.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - entity_reference_revisions
+    - node
+    - oe_content_entity_contact
+id: node.oe_publication_contact
+field_name: oe_publication_contact
+entity_type: node
+type: entity_reference_revisions
+settings:
```

Comment:
Is this ticket depends from EWPP-466?

Reply:
I don't think so, no such relation is described in EWPP-466.

Reply:
Since field.storage.node.oe_publication_contact.yml is in the EWPP-466, we will need to merge EWPP-466 first, then make rebase and update this PR.
BTW field.storage.node.oe_publication_contact.yml  in the EWPP-466 is multiple value field, but here it is single field. Based on the spec it has to be multiple. Or have we missed something?

Reply:
I updated this PR with the changes from EWPP-466. 

The task at https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EWPP-430 describes the **Contact** field with `cardinality:1` but I am not sure if the requirement is correct.